packages/orchestrator-ui-components/src/rtk/api.ts

```diff
@@ -48,6 +53,24 @@ export const handlePromiseErrorWithCallback = <T>(
     });
 };
 
+export const catchErrorResponse: ResponseHandler = async (
+    response: Response,
+) => {
+    if (
+        response.status < HttpStatus.Ok ||
+        response.status >= HttpStatus.MultipleChoices
```

Comment:
1st: this predicate is hard to follow ( "smaller than OK" or gte MultipleChoices ) -- consider extracting a predicate that is human readable, like "isNotSuccefsfulResponse".
2nd: in case of an error 401-unauthorized of 403-forbidden, wouldnt that trigger this if-case too? and thus never falling into the else-if and signing out the user?

Reply:
2nd: true, nice catch